images/win/scripts/Installers/Install-Nginx.ps1

```diff
@@ -0,0 +1,21 @@
+################################################################################
+##  File:  Install-Nginx.ps1
+##  Desc:  Install Nginx
+################################################################################
+
+# Stop w3svc service
+Stop-Service -Name w3svc | Out-Null
+
+# Install latest nginx in chocolatey
+$installDir = "c:\tools"
+Choco-Install -PackageName nginx -ArgumentList "--force", "--params", "/installLocation:$installDir /port:80"
```

Comment:
Since both IIS and Nginx lives on 80 port. I guess customers will have to disable IIS before starting Nginx or Apache. I wonder if it makes sense to mention it in documentation
@AlenaSviridenko what do you think?

Reply:
I think if the same behavior will happen on the local machine as well, then there is no need to mention it.